docs/make.jl

```diff
@@ -54,7 +54,7 @@ makedocs(
     doctest = ("fixdoctests" in ARGS) ? :fix : true,
     format = Documenter.HTML(canonical = "https://JuliaPhysics.github.io/SolidStateDetectors.jl/stable/", prettyurls = !("local" in ARGS)),

```

Comment:
... but not this?

We use the `"local"` keyword whenever building docs on our local computers.
Not sure what you intend with `"nonstrict"` instead of `"local"` here?

Reply:
I want to keep this consistent between our packages, I made the switch in the other packages as well. I think `nonstrict` is semantically clearer than `local`.

Reply:
The `local` in regard to `prettyurls` actually serves a different function, it's to build static HTML docs that don't need a local HTTP server to function properly (though now that VS-Code has such functionality out-of-the-box, we probably will use `local` less often).